src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -1,11 +1,133 @@
 import React from 'react';
+import Box from '../../../components/Box';
+import styled from 'styled-components';
+import {
+  Button,
+  Link,
+  FormControlLabel,
+  Checkbox,
+  FormControl,
+  FormLabel,
+  RadioGroup,
+  Radio,
+  Box as MuiBox,
+  Select,
+  InputLabel,
+} from '@material-ui/core';
+import { File } from '../../../types/license.types';
+import { KeyboardDatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import { useTranslation } from 'react-i18next';
+
+const StyledTitle = styled.div`
+  font-weight: bold;
+`;
+const StyledDescription = styled.div`
+  font-style: italic;
+`;
+const StyledActions = styled.div``;
+
+const StyledFormControl = styled(FormControl)`
+  width: 30%;
+  margin-top: 1rem;
+`;
+
+const StyledLicenseFormControl = styled(StyledFormControl)`
+  margin-top: 2rem;
+`;
 
 interface FileCardProps {
-  file: any;
+  file: File;
+  removeFile: () => void;
+  updateFile: (newFile: File) => void;
 }
 
-const FileCard: React.FC<FileCardProps> = ({ file }) => {
-  return <p>{file.name}</p>;
+const FileCard: React.FC<FileCardProps> = ({ file, removeFile, updateFile }) => {
+  const { t } = useTranslation('publication');
+
+  return (
+    <Box>
+      <StyledTitle>{file.name}</StyledTitle>
+      <StyledDescription>
+        {t('files_and_license.uploaded_size', { size: Math.round(file.data.size / 1000) })}
+      </StyledDescription>
+
+      <FormControlLabel
+        control={
+          <Checkbox
+            color="primary"
+            checked={file.administrativeContract}
+            onChange={() =>
+              updateFile({
+                ...file,
+                administrativeContract: !file.administrativeContract,
+              })
+            }
+          />
+        }
+        label={t('files_and_license.administrative_contract')}
+      />
+
+      {!file.administrativeContract && (
+        <MuiBox display="flex" justifyContent="space-between">
```

Comment:
MuiBox? kan vi ikke bruke en styled div her? eller vår Box?

Reply:
Vår box er bare en div med bakgrunnsfarge. Når MUI har komponenter med det vi trenger syns jeg det er bedre å bruke det enn å lage ting selv. Vi bør kanskje ta en diskusjon på dette også?

Reply:
jeg var skeptisk til styled components (pga vi putter css i kode), hvis vi nå skal begynne å dra inn det direkte i komponentene, så må vi ha en ny omgang. jeg synes personlig at det blir rotete og vanskelig å holde styr på hva som skjer hvor. MUI har jo komponenter for alt - det finnes jo MUIGrid også, men det er helt forferdelig å bruke IMO. kan diskutere dette også da, samtidig som vi diskuterer standardisering av tekstkomponenter